client/src/config/SimpleSchema.ts

```diff
@@ -0,0 +1,6 @@
+import SimpleSchema from "simpl-schema";
+
+// add the uniforms property to SimpleSchema
+SimpleSchema.extendOptions(['uniforms']);
```

Comment:
@wtrocki I created a config file for Simple Schema, so we can re-use this if we have other forms.

It wouldn't allow me to register the option globally in the router, so I went for this approach.

Reply:
Love it!